composer.json

```diff
@@ -55,8 +55,9 @@
     	"tao-extension-name" : "taoLti"
     },
     "require": {
-        "oat-sa/oatbox-extension-installer": "~1.1||dev-master"
-
+        "ext-openssl": "*",
+        "oat-sa/oatbox-extension-installer": "~1.1||dev-master",
+        "oat-sa/lib-lti1p3-core": "^2.3.0"
```

Comment:
let's keep in mind that we will update next sprint this

Reply:
Do we want to move to 3.0.0?

Reply:
Next sprint. There will be a task for this. Now we wont. :)